simtools/simulator.py

```diff
@@ -208,6 +213,12 @@ def _load_corsika_config_and_model(self, config_data):
             "corsika_parameters_file", None
         )
 
+        # Remove sim_telarray parameters from the CORSIKA config dictionary
+        # TODO - Replace this with a more elegant solution!
```

Comment:
I suggest creating issues for the TODOs, otherwise they get lost in the code (and pylint will probably complain)

Reply:
They will definitely not get lost in the code cause it's something I am working on actively. Generally I agree with you, I wouldn't have merged this to master with TODOs, but we decided to do it anyway just this time. Once I reach a state in which I can say the Rel 0 script is done I will remove them if they are still there and open issues.